src/main/java/org/example/parampammusic/repository/RoleRepository.java

```diff
@@ -0,0 +1,12 @@
+package org.example.parampammusic.repository;
+
+import org.example.parampammusic.entity.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface RoleRepository extends JpaRepository<Role, Long> {
```

Comment:
It seems, in Role entity your primary key is Integer. WHy do you have Long here?

Reply:
Not fixed